.editorconfig

```diff
@@ -12,7 +12,7 @@ indent_size = 2
 [*.md]
 trim_trailing_whitespace = false
 
-[*.cs, *.js, *.tsx]
+[*.{cs,js,tsx}]
```

Comment:
VSCode and Visual Studio didn't apply indentation correctly.
Turns out: This format has to be used.